require/require.go

```diff
@@ -0,0 +1,47 @@
+// Package require is a wrapper over testify's require package.
+package require
+
+import (
+	"github.com/stretchr/testify/require"
+)
+
+//go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 github.com/stretchr/testify/require.TestingT
+
+// Assertions adds assertions to testify's require lib.
+type Assertions struct {
+	*require.Assertions
```

Comment:
Same as the assert package, it is probably best to explicitly name this field

Reply:
same answer as above.